pkg/server/kube_version.go

```diff
@@ -12,8 +13,8 @@ func (s *Server) GetKubeVersion() string {
 		return s.KubeVersion
 	}
 	config := &rest.Config{
-		Host:      s.K8sProxyConfig.Endpoint.String(),
-		Transport: s.K8sClient.Transport,
+		Host:      s.K8sProxyConfigs["local-cluster"].Endpoint.String(),
```

Comment:
I'm not sure how this is used -- looks like for Helm -- but we probably need to pass the cluster name through.

Reply:
Yeah, this was more of just a bandaid until the Helm team gets their multicluster changes merged. I believe I addressed this with @dperaza4dustbit as he is working on the Helm stuff.